src/schema.graphql

```diff
@@ -84,13 +84,19 @@ type User {
   updatedAt: String
 
   friends: [User]! @relation(name: "FRIENDS", direction: "BOTH")
-  friendsCount: Int! @cypher(statement: "MATCH (this)<-[:FRIENDS]->(r:User) RETURN COUNT(r)")
+  friendsCount: Int! @cypher(statement: "MATCH (this)<-[:FRIENDS]->(r:User) RETURN COUNT(DISTINCT r)")
```

Comment:
@appinteractive while this *is* a way of solving the problem, I would prefer to have a unique constraint on the relationship. It creates less data and I believe it to be less error-prone. What about removing that relationship: Does that remove all matching friendship relationships?

Reply:
As far as I know, there are no uniqe constrains on relationships in Neo4j. I would chear if you could prove me wrong! 

But therefore I had added the todo in the description to: `"introduce new (un)shout and (un)follow mutations"` which will use `Merge` instead of `Create`. So we would just call ```
{
  follow(id: "u1", type: User)
}
```
and the same for `unfollow`, `shout` and `unshout`.

Reply:
I Added distinct mutation for that cases, but still we have inconsistant data from the alpha.